akka-http-core/src/test/scala/akka/http/model/UriSpec.scala

```diff
@@ -406,7 +418,7 @@ class UriSpec extends WordSpec with Matchers {
       normalize("http://sourceforge.net/projects/uriparser/") shouldEqual "http://sourceforge.net/projects/uriparser/"
       normalize("http://sourceforge.net/project/platformdownload.php?group_id=182840") shouldEqual "http://sourceforge.net/project/platformdownload.php?group_id=182840"
       normalize("mailto:test@example.com") shouldEqual "mailto:test@example.com"
-      normalize("file:///bin/bash") shouldEqual "file:///bin/bash"
+      normalize("file:///bin/bash") shouldEqual "file:/bin/bash"
```

Comment:
One more thought to maybe settle this issue:
We could change the rendering logic to simply format URIs with a *scheme* but an empty authority to always show the `//` delimiter. So `http:///foo` and `http:/foo` would still have the identical representation but always render to the former rather than the latter.
Maybe that's what you meant in the first place.